src/pyhf/paramview.py

```diff
@@ -0,0 +1,97 @@
+from . import get_backend, default_backend, events
+
+
+def index_helper(name, tensor_shape, batch_shape, par_map):
```

Comment:
any reason for this to be outside of the ParamViewer class?

Reply:
we can use anything except for the passed arguments. I tend to not include free functions in classes

Reply:
Right, so my only concern is that it's used once -- and only once. Is it planned to be used elsewhere? I agree with you on free functions, but single-use functions are weird to me.